SalesforceCommerceCloud/pwa-kit
# More localization fixes.

Follow up to #643 with a bit more work done.

* Recommended product tiles now use localized product names
* Wishlist Add to Cart button is now localized
* Wishlist toasts (added/removed) all use the same message for easier maintenance

# Description



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- Run the app locally. Choose a non-English locale. View a product, scroll to recommended products. Their names should not be in English.
- Run the app with pseudolocale. Add an item to your wishlist that needs no options before adding to cart (e.g. Striped Silk Tie). View your wishlist. Observe that "Add to Cart" in the wishlist gets localized.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
